### PR TITLE
Add dewpoint1-8 and dewpointin sensors

### PR DESCRIPTION
**Pull request by Entropy512**

### CODE REVIEW

File: custom_components/ecowitt/__init__.py [hunk 1]
```diff
@@ -274,6 +283,33 @@
...
                      "mdi:thermometer", 0),
+    TYPE_DEWPOINTINC: ("Indoor Dewpoint", TEMP_CELSIUS,
+                     TYPE_SENSOR, DEVICE_CLASS_TEMPERATURE,
+                     "mdi:thermometer", 0),
+    TYPE_DEWPOINT1C: ("Dewpoint 1", TEMP_CELSIUS,
+                     TYPE_SENSOR, DEVICE_CLASS_TEMPERATURE,
+                     "mdi:thermometer", 0),
+    TYPE_DEWPOINT1C: ("Dewpoint 2", TEMP_CELSIUS,
+                     TYPE_SENSOR, DEVICE_CLASS_TEMPERATURE,
+                     "mdi:thermometer", 0),
```
Comment (garbled1) [inline custom_components/ecowitt/__init__.py:294]: Oops, you repeated 1C over and over here..  :)

Comment (Entropy512) [reply custom_components/ecowitt/__init__.py:294]: whoops, will fix and re-push tonight.

Comment (Entropy512) [reply custom_components/ecowitt/__init__.py]: Re-pushed the commit with changes

**Comment by garbled1** [comment]
Thanks!

**Pull request merged**